### PR TITLE
Add Thrust/CUB 1.12.1 and 1.13.0.

### PR DESCRIPTION
**Pull request by allisonvacanti**
Adding new versions of these CUDA libraries. See also compiler-explorer/infra#538

### CODE REVIEW

**Comment by partouf** [comment]
The new versions should be added to https://github.com/compiler-explorer/compiler-explorer/pull/2747/files#diff-b7344ac590742f9fcdee51b23c97652bbb31c63b1ef2fc02df4d4dd186659a95R111

File: etc/config/cuda.amazon.properties [hunk 1]
```diff
@@ -122,6 +122,10 @@ libs.thrustcub.versions.111000.version=1.11.0
 libs.thrustcub.versions.111000.path=/opt/compiler-explorer/libs/thrustcub/1.11.0:/opt/compiler-explorer/libs/thrustcub/1.11.0/dependencies/cub
 libs.thrustcub.versions.112000.version=1.12.0
 libs.thrustcub.versions.112000.path=/opt/compiler-explorer/libs/thrustcub/1.12.0:/opt/compiler-explorer/libs/thrustcub/1.12.0/dependencies/cub
+libs.thrustcub.versions.112000.version=1.12.1
```
Comment (partouf) [inline etc/config/cuda.amazon.properties:125]: version id should probably be 112100?

File: etc/config/cuda.amazon.properties [hunk 2]
```diff
@@ -122,6 +122,10 @@ libs.thrustcub.versions.111000.version=1.11.0
...
+libs.thrustcub.versions.112000.path=/opt/compiler-explorer/libs/thrustcub/1.12.1:/opt/compiler-explorer/libs/thrustcub/1.12.1/dependencies/cub
+libs.thrustcub.versions.112000.version=1.13.0
```
Comment (partouf) [inline etc/config/cuda.amazon.properties:127]: version id should probably be 113000

**Review by partouf — CHANGES_REQUESTED** [review]

**Comment by allisonvacanti** [comment]
D'oh -- should be all set now.

**Pull request merged**